build.sbt

```diff
@@ -2,8 +2,8 @@ inThisBuild(Seq(
   name := "Scala DOM Types",
   normalizedName := "domtypes",
   organization := "com.raquo",
-  scalaVersion := "2.12.9",
-  crossScalaVersions := Seq("2.11.12", "2.12.9")
+  scalaVersion := "2.12.8",
```

Comment:
```suggestion
  scalaVersion := "2.12.9",
```

Reply:
Ah yeah, I keep switching that value to 2.12.8 for metals.

Reply:
haha, same for me :wink:, metals has 2.12.9 support now on master. You can use the latest snapshot.